### DISCUSSION TITLE
Limited ECS_CONTAINER_CONTEXT configuration to run tasks in a different cluster from daemon

### DISCUSSION THREAD

**Discussion by srujithpoondla03** [dagster-io/dagster#25069] [Q&A]
Is there a specific reason on why ECS_CONTAINER_CONTEXT is limited to only part of the ECS configuration like cluster, network_configuration etc.. are not available? The use case I am looking for is to not use current ecs task config rather I would like to pass the cluster and network configuration from user code through container context instead of a specific task definition arn.

**Comment by garethbrickman** [comment]
[Reply from in the Community Slack](https://dagster.slack.com/archives/C01U954MEER/p1727963407040489?thread_ts=1727887102.299339&cid=C01U954MEER):
>My assumption is that these parameters are restricted because they require specific values in order for Dagster to work correctly. I don't know if this is 100% true though, but changing network configuration could certainly prevent Dagster from being able to communicate with the task. The task also needs to have Dagster installed, and will need to have its entrypoint overridden by Dagster to get the user code launched correctly. So I think they've introduced some restrictions so users don't shoot themselves in the foot trying to launch arbitrary ECS tasks as runs

Feel free to open a PR for this!
